CMakeLists.txt

```diff
@@ -469,7 +469,7 @@ ADD_EXECUTABLE(qc ${QC_SRC})
 # SDL2 Dependency (Detection pulled from FAudio)
 if (DEFINED SDL2_INCLUDE_DIRS AND DEFINED SDL2_LIBRARIES)
 	message(STATUS "using pre-defined SDL2 variables SDL2_INCLUDE_DIRS and SDL2_LIBRARIES")
-	target_include_directories(qc PUBLIC "$<BUILD_INTERFACE:${SDL2_INCLUDE_DIRS}>")
+	target_include_directories(qc PUBLIC ${SDL2_INCLUDE_DIRS})
```

Comment:
I'd like to know why this change was needed as well - this comes straight from FAudio which Debian packages, so as far as I know this should work. I ran into an issue with the includes myself, so there _is_ a difference needed but I'd like to know what it really is:

https://github.com/blendogames/quadrilateralcowboy/blob/master/idlib/precompiled.h#L122

Reply:
I haven't been able to figure this one out, and have been able to build from a clean checkout without this change in place at all, so I've reverted it.